src/app/services/postCreatePolicyRole.js

```diff
@@ -4,8 +4,8 @@ const logger = require("../../infrastructure/logger");
 const validate = async (req) => {
   const model = {
     appId: req.params.sid || "",
-    roleName: req.body.roleName || "",
-    roleCode: req.body.roleCode || "",
+    roleName: req.body.roleName.trim() || "",
```

Comment:
Really minor, but can you write a test for this?  We're intentionally choosing to strip the whitespace so we can 'document' that choice via the tests so that doesn't change without another intentional choice in the future